test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -463,3 +464,14 @@
         (is (= handled-response (wrapped-handler put-request)))
         (is (= handled-response (wrapped-handler post-request)))
         (is (= handled-response (wrapped-handler delete-request)))))))
+
+(deftest json-response-test
```

Comment:
I agree that it's preferable to have much more test coverage than this.  If we're just cut/pasting code over here, though, I'm probably okay with not requiring new tests to be written as a part of this PR.